charts/posthog/values.yaml

```diff
@@ -832,6 +832,14 @@ clickhouse:
   user: admin
   # -- Clickhouse password
   password: a1f31e03-c88e-4ca6-a2df-ad49183d15d9
+  # -- Clickhouse password value from an existing secret that needs to be in
+  # the namespace where posthog is deployed into. Will not use the above
+  # password value if set
+  existingSecret: {}
```

Comment:
Thanks, just updated in [7c59a51](https://github.com/PostHog/charts-clickhouse/pull/506/commits/7c59a512aa6bf4285649cf7106f6ccdc329da744)

Reply:
Ah sorry, if you could make them just empty strings this should generate the correct docs.

Reply:
👍 that makes a lot more sense indeed, the `{}` was more appropriate for the other way
[a663f99](https://github.com/PostHog/charts-clickhouse/pull/506/commits/a663f9937158f864d0b4f35148927576bb6958d9)

Let me know if I can/should squash the commits or rename them

Reply:
Repo is set to squash and merge. If you want to craft a commit message specifically then feel free, but it's fine to just leave and it will be a single commit with all the commit messages concatenated.